### PR TITLE
updated probe-rs-tools instructions

### PR DESCRIPTION
**Pull request by BartMassey**
See #579. Maybe not the ultimate solution, but should at least triage the situation.

### CODE REVIEW

File: microbit/src/03-setup/README.md [hunk 1]
```diff
@@ -69,8 +69,14 @@ cargo-size 0.3.3
 In order to install cargo-embed, first install its [prerequisites](https://probe.rs/docs/getting-started/installation/) (note: these instructions are part of the more general [`probe-rs`](https://probe.rs/) embedded debugging toolkit). Then install it with cargo:
 
 ```console
-$ cargo install probe-rs-tools --vers 0.24.0
+$ cargo install probe-rs-tools --vers '^0.24'
+```
+
+**NOTE** This may fail due to frequent changes in `probe-rs`. If so, go to <https://probe.rs> and follow the current installation instructions there.
 
+Finally, verify that you have successfully installed `cargo-embed`.
```
Comment (eldruin) [inline microbit/src/03-setup/README.md:77]:
```suggestion
Finally, verify that you have successfully installed `cargo-embed` by running:
```

**Review by eldruin — COMMENTED** [review]
Thanks! just added a small improvement.

**Comment by BartMassey** [comment]
Turns out `--locked` resolves the issue. I'll update the PR.

**Review by eldruin — APPROVED** [review]
Thank you!

**Pull request merged**